WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/NotificationsUtils.java

```diff
@@ -87,7 +87,7 @@ public static void getPushNotificationSettings(Context context, RestRequest.List
         if (!TextUtils.isEmpty(deviceID)) {
             settingsEndpoint += "?device_id=" + deviceID;
         }
-        WordPress.getRestClientUtilsV1_1().get(settingsEndpoint, listener, errorListener);
+        WordPress.getRestClientUtilsV1_1().getWithLocale(settingsEndpoint, listener, errorListener);
```

Comment:
I've renamed the existing `get` method to `getWithLocale` because it adds a `locale` query parameter to every request. As you can see in this PR, some of our endpoints use `_locale` or even `lang` query parameters, so I've renamed it and created a `get` method that doesn't add any locale query parameter automatically.